app/controllers/deploys_controller.rb

```diff
@@ -49,6 +49,86 @@ def recent
     end
   end
 
+  # little search endpoint.  
+  # takes the same tokens that are used by the filter stuff
```

Comment:
not sure what that filter stuff is, can you use a real word instead ? maybe link to it ?